pahfit/feature_strengths.py

```diff
@@ -53,3 +55,54 @@ def line_strength(ampl, mean, stddev):
     strength = ((np.sqrt(np.pi / np.log(2))) * const.c.to('micron/s').value / 2) * (ampl * fwhm / mean**2) * 1e-26
 
     return strength
+
+
+def featcombine(ftable):
+    """
+    Combine dust features strengths.
+    Parameters
+    ----------
+    ftable : data table
+        Astropy table containing lines.
+    Returns
+    -------
+    cftable : data table
+        Astropy table containing combined dust feature strengths.
+    """
+    # Define combined strength dictionary.
+    cfdic = {'PAH_62': {'range': [6.2, 6.3]},
+             'PAH_77_C': {'range': [7.3, 7.9]},
+             'PAH_83': {'range': [8.3, 8.4]},
+             'PAH_86': {'range': [8.6, 8.7]},
+             'PAH_112_C': {'range': [11.2, 11.4]},
```

Comment:
Copy from PR #142:
Note that in PAHFIT the combined PAH feature at 11 µm is dubbed "11.3 complex". For consistency, might be better to name it "PAH_113_C". If we decide to stick with "PAH_112_C", will need to mention it somewhere to avoid confusion.

Reply:
Agree.